README.md

```diff
@@ -1,5 +1,11 @@
 # simple-react-calendar <sup>[![Version Badge][npm-version-svg]][package-url]</sup>
 
+---
+
+## ⚠️ Please note that the repository is no longer maintained and was archived. If you need to use a calendar, please use the `Calendar` component from `@toptal/picasso` package.
```

Comment:
Maybe it's worth pointing to other feasible alternatives? Mainly for consumers who do not use Picasso, we could point to `react-day-picker` as a suggested alternative

Reply:
@mkrl Added, thank you!